### PR TITLE
second try

### PR DESCRIPTION
**Pull request by ghost**
skdjhgjwlehg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # branch_testing
 branching demo
 
-The irreducible nature of the prime contruct are inconcievable to feeble mind. Whatevs lol. 
-ffffff
\ No newline at end of file
+second try
```
Comment (kingbsolomon) [inline README.md:4]: have you thought about using 2nd instead of second?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # branch_testing
 branching demo
 
-The irreducible nature of the prime contruct are inconcievable to feeble mind. Whatevs lol.
```
Comment (kingbsolomon) [inline README.md:4]: this is a comment

**Comment by kingbsolomon** [comment]
mic check 1,2 1,2

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:4]: that is a good idea. automation of lang is desirable.

**Review by ghost — COMMENTED** [review]
blah

**Review by kingbsolomon — APPROVED** [review]

**Pull request merged**